nodejs/README.md

```diff
@@ -0,0 +1,15 @@
+socrata/nodejs
+
+socrata/base image with nodejs(0.10.25) and npm installed natively and the latest
+of io.js (2.0.0) and node (0.12.2) installed in userspace with n, a node package 
```

Comment:
I'm a bit wary of specifying the version numbers in these comments since it doesn't seem like we are actually enforcing that it is these versions.